kroxylicious-test-tools/src/main/java/io/kroxylicious/test/client/KafkaClientHandler.java

```diff
@@ -47,15 +60,28 @@ public void channelReadComplete(ChannelHandlerContext ctx) {
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
-        cause.printStackTrace();
```

Comment:
Do the exceptions still bubble up somewhere or is this silently ignoring them?

Reply:
This is Netty's inbound exception handling.  My motivation for removing the printStackTrace was incorrect.  I wrongly believed I would see exception related to the close of the channel from the server side.  That's not correct, the channel's closure is graceful and there's no exception.

I'll change the printStackTrace into a warn.  We don't expect to follow this path.